### ISSUE TITLE
Disable GoogleAnalytics when not running in production

### ISSUE THREAD

**Issue by andretti1977** [appernetic/hugo-nederburg-theme#45]
As in the title, it would be a good thing to disable GoogleAnalytics when not running in production.
There are several ways to do this but the one i prefer is to check for the env var HUGO_ENV.
[Here](https://discourse.gohugo.io/t/how-to-exclude-google-analytics-when-running-under-hugo-local-server/6092/7) user budparr suggests this code:
```
{{ if eq (getenv "HUGO_ENV") "production" }}
        {{ template "_internal/google_analytics_async.html" . }}
{{ end }}
```
You should simply change layouts/partials/head.html. I would do a pull request but since there are many ways to disable it and since that would force your user to provide that env var, i prefer you to solve the issue in the way you prefer

**Comment by appernetic** [comment]
Yes, I agree. I thought that there was a check for localhost in the internal Hugo file?
